doc/ref_model/chapters/chapter07.md

```diff
@@ -419,7 +416,7 @@ Poor code quality is a factor of risk. Open source code advantage is its transpa
 
 Vulnerability management must be continuous: from development to runtime, not only on the development process, but during all the life of the application or workload or service. When a public vulnerability on a component is released, the update of the component must be triggered. When an SBOM recording the code composition is provided, the affected components will be easier to identify. It is essential to remediate the affected components as soon as possible, because code transparency can also be exploited by attackers who can take the benefit of vulnerabilities.
 
-The CVE must be used to identify vulnerabilities and their severity rating. CVE identifies, defines, and catalogues publicly disclosed cybersecurity vulnerabilities.
+The CVE and the CVSS must be used to identify vulnerabilities and their severity rating. The CVE identifies, defines, and catalogues publicly disclosed cybersecurity vulnerabilities. The CVSS specifies an open framework to calculate vulnerabilities severity rate.
```

Comment:
```suggestion
The CVE and the CVSS must be used to identify vulnerabilities and their severity rating. The CVE identifies, defines, and catalogues publicly disclosed cybersecurity vulnerabilities while the CVSS is an open-source framework to calculate the vulnerabilities' severity rating.
```

Reply:
For readability, I am suggesting a compound sentence instead of two individual sentences.

Reply:
@pgoyal01 Thanks, suggestion added